Linked list with void pointers.c

```diff
@@ -5,29 +5,26 @@ typedef struct node
 {
     void *data;
     struct node *next;
+    int (*funptr)();

```

Comment:
What is funptr? What does it do. use proper meaningfull names, and why does it return int?

Reply:
Function pointer annayha

Reply:
What is its purpose, It should be renamed to getSize i guess. and as getSize you are storing in node itself, you can store display_datatype also in node i guess. so with that we can do multiple datatype linked lists.

Reply:
I am storing display_datatype function pointer in *funptr annayha

Reply:
it is not meaningfull .

Reply:
not meaningfull enough.